app/views/dashboard/student.haml

```diff
@@ -8,32 +8,26 @@
       %li Your #{interview.application_submission.department.name} interview
       has been scheduled for #{interview.information}.
     .interview_review_link
-      = link_to 'Review your application', interview.application_submission
+      = link_to 'View your application', interview.application_submission
 %h2 Available Applications
 Please select the application which you would like to submit.
 - @positions.each do |department, positions|
   %h3= department.name
   - positions.each do |position|
-    - app_submitted = @application_submissions.try(:keys).try(:include?, position)
-    - if app_submitted
-      // most recent application, if more than one
-      - application = @application_submissions[position].last
-      - should_show = application.pending? || should_show_denied_applications?
-    - if app_submitted && should_show
-      %li= link_to "Review your application for #{position.name}", application
-    - if position.application_template.try(:active?)
-      - if app_submitted
-        - if should_allow_resubmission?
-          -if application.reviewed
+    // most recent application, if more than one
+    - if application = @application_submissions.try(:[], position)&.last
```

Comment:
Two things:

1. Rubocop would suggest (and I agree) that there should be parentheses here to indicate that you intended to do an assignment inside a conditional
2. There are a few ways to do this without `try`:

    ```ruby
    @application_submissions&.dig(position)&.last
    @application_submissions&.fetch(position, nil)&.last
    @application_submissions&.[](position)&.last #Look at majestic string of punctuation
    ```